### PR TITLE
add : Artworks routes inside backend

### PR DESCRIPTION
**Pull request by Jhanemba64**

### CODE REVIEW

File: backend/database/schema.sql [hunk 1]
```diff
@@ -1,4 +1,47 @@
-create table item (
-  id int unsigned primary key auto_increment not null,
-  title varchar(255) not null
+
+CREATE DATABASE IF NOT EXISTS mapping_art_db;
+USE mapping_art_db;
+
+CREATE tABLE user (
```
Comment (Lambseb) [inline backend/database/schema.sql:5]: full upper case TABLE

**Review by Jubdc — APPROVED** [review]

File: backend/src/tables.js [hunk 2]
```diff
@@ -4,10 +4,12 @@
...
 const ItemManager = require("./models/ItemManager");
+const ArtworkManager = require("./models/ArtWorkManager");
 
 const managers = [
   ItemManager,
   // Add other managers here
+  ArtworkManager,
 ];
 
 // Create an empty object to hold data managers for different tables
```
Comment (Lambseb) [inline backend/src/tables.js:15]: remove inline comment

**Review by Lambseb — APPROVED** [review]

File: backend/src/models/ArtWorkManager.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    const [result] = await this.database.query(
+      `INSERT INTO ${this.table} (title, adress, validated, categories_id) VALUES (?, ?, ?, ?)`,
+      [title, adress, validated, categorieID]
+    );
+
+    return result.insertId;
+  }
+
+  async update(id, title, adress, validated, categorieID) {
+    // console.log(id, title, adress, validated, categorieID);
```
Comment (Kino-ki) [inline backend/src/models/ArtWorkManager.js:32]: remove console.log

**Review by Kino-ki — CHANGES_REQUESTED** [review]

**Review by Jubdc — APPROVED** [review]

**Review by Jubdc — APPROVED** [review]

File: backend/src/models/ArtWorkManager.js (hunk 3 above)
Comment (Jhanemba64) [reply backend/src/models/ArtWorkManager.js]: Done !

**Pull request closed without merging**